website/docs/components/link/inline/partials/guidelines/guidelines.md

```diff
@@ -14,7 +14,9 @@
 
 The Inline Link is only published as a component in Ember, there is no corresponding component in Figma. To achieve the same results, apply **text** and **color** styles from the [Foundations library](https://www.figma.com/file/oQsMzMMnynfPWpMEt91OpH/HDS-Product---Foundations?type=design&node-id=2130%3A2&t=6XBReWOxMRTiW1Iv-1) to your text.
 
-The Ember component inherits the styles of the text around it. Depending on the surrounding text, the link style in Figma may need to be overridden to ensure that the link and body text are the same.
+The Ember component inherits the typographic styles of the text around it (font, weight, etc), while the [color](#color) is determined by the `@color` argument.
+
+Since a component isn’t available for this within Figma, you will need to manually adjust the style of the text. To do this, select the text needed for the link, and change the decoration to "Underline".
```

Comment:
Actually, this whole section is now a little odd because there are additional instructions above about achieving this in Figma. Maybe we should take a more holistic look at this section and provide more clarity around how to accomplish this within Figma and update the content to make it flow better. There's already guidance in the stickersheet...maybe you could reuse some of that content?

Reply:
This was very closely mimicking the content in the stickersheet previously, let me dig in a bit more to see if I can make it flow a bit better.

Reply:
Sorry! I thought there was more specific information about applying the styles, but you're right.